db/README.md

```diff
@@ -9,7 +9,6 @@
 In the base project directory, the following commands can be run to get the bike data loaded:
 ```
 yarn build:db
-yarn db:seed
```

Comment:
I've removed this step because It appears that the database is already seeded [in the the Dockerfile](https://github.com/omegabytes/lane_breach/blob/master/db/Dockerfile#L11-L12).

Reply:
No unfortunately it doesn't seed the db in the dockerfile (I think). It simply downloads the the bikeway network shape file and converts it into a sql file using postgis. 

I kind of abuse Dockerfile for this b/c to convert the shapefile to sql requires postgis to be installed and ideally I dont' want devs to have to install any dependencies (aside from docker) to get the stack up. Ideal would be to run `psql bikelanes bikelanes -f /data/bikeway_networks.sql'` in the Dockerfile but I don't believe psql is running at that time. FWIW, I haven't looked into seeding via a dockerfile.

In any case, I think the change you made for `yarn db:seed` to seed using the rails seed file is the right move and we should remove lines 9-12 [here](https://github.com/omegabytes/lane_breach/blob/5280d938ff5aafecbd0cb5dcc5dc905032465653/db/Dockerfile#L9), and just take the sql file and check it into /lanebreach-api/db/ and then in the `seeds.rb` we can ingest it. Anyways hopefully that makes sense. We can talk more wed!